src/Components/Onboarding/Types.ts

```diff
@@ -2,7 +2,7 @@
  * The props interface that the step needs to implement for the wizard.
  */
 export interface StepProps {
-  onNextButtonPressed: (increaseBy) => void
+  onNextButtonPressed: (increaseBy?) => void
```

Comment:
Making the increasedBy parameter optional since we have set the default value as 1.